tests/models/plants/test_plants_model.py

```diff
@@ -161,6 +161,27 @@ def test_PlantsModel_estimate_gpp(fxt_plants_model, fixture_core_components):
     )
 
 
+def test_PlantsModel_allocate_gpp(fxt_plants_model, fixture_core_components):
+    """Test the allocate_gpp method."""
+
+    fxt_plants_model.stem_gpp = {
+        cell_id: np.array([55]) for cell_id in fxt_plants_model.communities.keys()
+    }
+    # Allocate GPP
+    fxt_plants_model.allocate_gpp()
+
+    for community in fxt_plants_model.communities.values():
+        # .. TODO: not sure what to test here? Main thing it does is change the
```

Comment:
Ha! The `.. TODO::` annotation is specific to function docstrings, so they appear nicely styled in the HTML docs. Here, plain old `TODO` will cover it 😄